### PR TITLE
Tickets/dm 43463   base and summit deploy

### PR DESCRIPTION
**Pull request by womullan**

### CODE REVIEW

File: summit.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Create Schema
+-------------
+For consitancy the schema file used at the base generated from Felis and post processed with KT's script was used.
+
+With Summit VPN connected:
+
+.. code-block::
+
+  psql  -h postgresdb01.cp.lsst.org -U oods butler 
+  butler=> \i summit-lattis.sql
```
Comment (ktlim) [inline summit.rst:18]:
```suggestion
  butler=> \i summit-latiss.sql
```

File: usdf.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Deployment at USDF
+==================
+First the values file in Phalanx was created namely 'values-usdfprod.yaml <https://github.com/lsst-sqre/phalanx/blob/162905690a25309bc916f1304fc785c472f0e9ae/applications/consdb/values-usdf.yaml>'
+
+'Secret was created  in the USDF vault <https://vault.slac.stanford.edu/ui/vault/secrets/secret/show/rubin/usdf-rsp/consdb>' with values from base.
```
Comment (ktlim) [inline usdf.rst:5]:
```suggestion
Secret was created  in `the USDF vault <https://vault.slac.stanford.edu/ui/vault/secrets/secret/show/rubin/usdf-rsp/consdb>` with values from base.
```

File: usdf.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Deployment at USDF
+==================
+First the values file in Phalanx was created namely 'values-usdfprod.yaml <https://github.com/lsst-sqre/phalanx/blob/162905690a25309bc916f1304fc785c472f0e9ae/applications/consdb/values-usdf.yaml>'
```
Comment (ktlim) [inline usdf.rst:3]:
```suggestion
First the values file in Phalanx was created namely `values-usdfprod.yaml <https://github.com/lsst-sqre/phalanx/blob/63377b6ae3d8661589ebf4c019232982785c8ec6/applications/consdb/values-usdfprod.yaml>`
```

File: summit.rst [hunk 4]
```diff
@@ -0,0 +1,24 @@
+Deployment at Summit
+====================
+Set consdb to true in the phalanx environments values-summit.yaml.
+
+Created values file in Phalanx 'values-summit.yaml <https://github.com/lsst-sqre/phalanx/blob/162905690a25309bc916f1304fc785c472f0e9ae/applications/consdb/values-usdf.yaml>'
```
Comment (ktlim) [inline summit.rst:5]:
```suggestion
Created values file in Phalanx `values-summit.yaml <https://github.com/lsst-sqre/phalanx/blob/63377b6ae3d8661589ebf4c019232982785c8ec6/applications/consdb/values-summit.yaml>`
```

**Review by ktlim — APPROVED** [review]

File: felis.rst [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Felis puts quotes on names which make them case sensitive in postgress - we remove those with:
+
+.. code-block::
+
+   yml/post.sh  tmp.sql  >> summit-latiss.sql
+
+
+Now we have the schema file we can use in any of the DBs.
+
+
```
[the remark this answers is not in the document]
Comment (womullan) [reply felis.rst:36]: the first line of the generated sql is  drop schema cdb_latiss cascade; create schema cdb_latiss;

**Pull request merged**